### PR TITLE
Iterate on styles

### PR DESCRIPTION
**Pull request by patrickberkeley**
@jordpo

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    </div>
+  </div>
+</div>
+
+<script src="https://code.jquery.com/jquery-1.12.0.min.js"></script>
+<script>window.jQuery || document.write('<script src="js/vendor/jquery-1.12.0.min.js"><\/script>')</script>
+<script src="js/plugins.js"></script>
+<script src="js/main.js"></script>
+
+<!-- Google Analytics: change UA-XXXXX-X to be your site's ID. -->
```
Comment (jordpo) [inline _includes/footer.html]: do we have an account setup?

File: _sass/components/_layout.scss [hunk 2]
```diff
@@ -63,30 +63,28 @@ $props: (
...
-        @if #{$side} == 'all' {
-          #{$prop}: #{$i * 10}px !important;
-        } @else if #{$side} == 'left-right' {
-          #{$prop}-left: #{$i * 10}px !important;
-          #{$prop}-right: #{$i * 10}px !important;
-        } @else if #{$side} == 'top-bottom' {
-          #{$prop}-top: #{$i * 10}px !important;
-          #{$prop}-bottom: #{$i * 10}px !important;
-        } @else {
+        #{$prop}-#{$side}: #{($i * 10)/2}px !important;
```
Comment (jordpo) [inline _sass/components/_layout.scss]: why divide by 2?

**Comment by jordpo** [comment]
LATM!

File: _sass/components/_layout.scss (hunk 2 above)
Comment (patrickberkeley) [inline _sass/components/_layout.scss]: To scale down for smaller screens

File: _includes/footer.html (hunk 1 above)
Comment (patrickberkeley) [inline _includes/footer.html]: We do now. Pushed

**Pull request merged**